src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -89,8 +81,16 @@ public PersonBuilder withEmail(String email) {
         return this;
     }
 
+    /**
+     * Sets the {@code Email} of the {@code Person} that we are building.
```

Comment:
Should this be "{@ MatriculationNumber}" instead or "Email"?

Reply:
Ahh I see. Thanks!